miro_preprocessor/README.md

```diff
@@ -0,0 +1,7 @@
+# miro_preprocessor
+
+This stack is responsible for sorting Miro metadata and assets into one of three piles:
+
+*   Cold store / Amazon Glacier
+*   Tandem Vault
+*   The Catalogue API
```

Comment:
I think a brief explanation of the different pieces might be useful.

e.g. What’s the role of `xml_to_json_converter`?